CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and **Merged pull requests**. Critical items to know are:
 The versions coincide with releases on pypi.
 
 ## [0.0.x](https://github.com/singularityhub/singularity-compose/tree/master) (0.0.x)
+ - adding ability to specify a different log folder for each instance
  - resolv.conf, etc.hosts generated if needed, network disabled non-sudo users (0.0.17)
```

Comment:
Okay this is a great start! What is missing is an bump to version.py, and then this is added in parens to the changelog. I'd also like to see documentation added to guide the user for how to define a logging folder - and since we are changing the recipe parameters we should discuss if this warrants a minor version change (I think possibly so) and in which case we need to discuss how to represent that in the documentation, and how to check in the client.

Reply:
I just updated the documentation, the short paragraph I added should be sufficient as the parameter is very simple and - at the same time - quite secondary (considering the "importance" of other parameters such as volumes). 

I guess such a small change could trigger at most a minor version change, what's your feeling about that?

Reply:
One quick question - have you tested this? The choice of the log folder (and naming) is determined by Singularity - the client here simply generates the default path, and then forwards to the client (if it exists). Is your use case that you've changed the default Singularity logs folder, and it needs to also be changed for the client? If you just change the log folder here without any changes to Singularity, it won't be the case that the logs change location. Singularity compose is not writing the logs, but rather, Singularity is.

Reply:
I didn't take this into account: I tested the behavior of set-get by using the `logs` param but I didn't actually realize that there was no option in singularity to do so.

What about creating symbolic links to the singularity default folder in case `logs` is set?

Reply:
Singularity compose isn’t writing a file... I think if you really want to tackle this you should start with the Singularity software first.

Reply:
See how path was solidified at https://github.com/sylabs/singularity/issues/1115. I'm not aware that's it's possible to customize the logs folder but it doesn't hurt to ask!